bccsp/schemes/aries/issuer.go

```diff
@@ -58,6 +58,32 @@ type Issuer struct {
 	Curve *math.Curve
 }
 
+func (i *Issuer) Bases(key types.IssuerPublicKey, ipkType types.CommitmentBasesRequest, RhIndex, EidIndex, SKIndex int) (map[types.CommitmentType]interface{}, error) {
```

Comment:
please, add a comment here describing what will be returned

Reply:
done, thanks!